src/lib/common/errorMessages.h

```diff
@@ -98,4 +98,9 @@
 
 #define ERROR_DESC_BAD_VERB                           "method not allowed"
 
+
+#define ERROR_NOTIMPLEMENTED                          "NotImplemented"
+#define ERROR_IDPATTERN_NOTSUPPORTED                  "Unsupported idPattern"
+#define ERROR_TYPEPATTERN_NOTIMPLEMENTED              "typePattern is not supported"
+
```

Comment:
Looking to other cases in errorMessages.h I think names should be:

ERROR_IDPATTERN_NOTSUPPORTED -> ERROR_DESC_IDPATTERN_NOTSUPPORTED
ERROR_TYPEPATTERN_NOTIMPLEMENTED -> ERROR_DESC_ERROR_TYPEPATTERN_NOTIMPLEMENTED

Reply:
Fixed in 5d4d5aeb0